app/models/user.rb

```diff
@@ -52,7 +52,10 @@ class User < ApplicationRecord
 
   validates :first_name, presence: true
   validates :last_name, presence: true
-  validates :email, presence: true, uniqueness: {scope: :organization_id}
+  validates :email, presence: true, uniqueness: {scope: :organization_id}, format: {
+    with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i,
+    message: "should be a valid email address"
```

Comment:
nice! can we add the validation message to the en.yml locale file? You will see how we have handled this for all other active record validations. 

Also could we please make the message `Invalid email` to keep it nice and brief?